nft/nfts.yaml

```diff
@@ -26,7 +26,7 @@ paths:
         - $ref: nfts_body.yaml#/pageKey
           description: 'String - Cursor key for pagination. If more results are available, a pageKey will be returned in the response. Pass back that pageKey as a param to fetch the next 100 NFTs.'
         - $ref: nfts_body.yaml#/pageSize
-          description: 'String - Number of NFTs to be returned per page. Defaults to 100. Max is 100.'
+          description: 'String - Number of NFTs to be returned per page. Defaults to 100. Max is 100. NOTE: This parameter is not supported on the Rinkeby testnet.'
```

Comment:
rather than saying what it's **not** supported on we should say what networks it is supported on. As a user after reading this my next thought is "ok so what networks can i use this on?"